package.json

```diff
@@ -29,7 +29,7 @@
     "queryResource": "query",
     "okapiInterfaces": {
       "cancellation-reason-storage": "1.1",
-      "circulation": "10.0 11.0",
+      "circulation": "10.0 11.0 12.0",
```

Comment:
Looking at the changes to the shape of the request data where `item` has been replaced by `instance`, do we really continue to support versions `10.0` and `11.0`? It feels like we should instead support `12.0` exclusively and bump the module's major version.

Reply:
Yeah, you right, I think we should use only 12 version. Because after migration previous versions will send invalid responce.

Reply:
Done. Could you please clarify, if we affect only `requests` module, should we leave last three versions in other modules, or only `12.0` as here?

Reply:
It's up to you whether to continue supporting older interfaces in other modules. Here, we were clearly adding v12-specific code. If other modules that use the circulation interface don't do anything that is v12-specific, there is no reason to drop support for those version.

Reply:
Ok. I think we can just add `12.0` in other modules, because we have breaking changes only for `request` module.
Can we count this conversation as resolved?
And could you please approve this PR, if now everything looks good for you.
Thanks!